### ISSUE TITLE
3.1.14 - Search (real) page number and display (Content view)

### ISSUE THREAD

**Issue by janahoffmann** [gbhl/bhl-europe#96]
Enter page number in box
Display only requested page.

Precondition: correct pagination or image numbers

**Comment by leenamba** [comment]
@jannahoffmann
Assumes we're in the book viewer only enter the page number and not a page range.  
If "real" page numbers are not provided in the metadata image numbers will be used

**Comment by janahoffmann** [comment]
@leenamba
Yes. This feature is related to the book viewer only. Like in the Acrobat Reader version there should be a box where you can enter the page you want to go to...I agree, if no pagination is provided this will lead to image numbers...still you have an idea where you are in the document (in relation). Description is updated.
